gcn_classic_text_to_json/notices/amon/README.md

```diff
@@ -0,0 +1,28 @@
+# AMON Text Conversion
+
+Parses through all webpages with AMON text notices and creates a JSON with GCN schema keywords. Creates a `amon_jsons` directory inside an `output` directory and saves jsons as `AMON_{serial_number}_{record_number}.json` where serial_number is a random iterating number with no association to the notices and record_number is the current notice in the webpage.
+
+### Uses the following fields from the core schema for text notice fields
+- `id` &#8594; EVENT_NUM, RUN_NUM
+- `ra` &#8594; SRC_RA
+- `dec` &#8594; SRC_DEC
+- `ra_dec_error` &#8594; SRC_ERROR
+- `alert_datetime` &#8594; NOTICE_DATE
+- `trigger_time` &#8594; DISCOVERY_DATE, DISCOVERY_TIME
+- `event_name` &#8594; EVENT_NAME
+- `record_number` &#8594; REVISION
+- `far` &#8594; FAR
+
+### Defines the following new fields for the text notice fields
+- `notice_type` &#8594; NOTICE_TYPE
```

Comment:
`NOTICE_TYPE` is quiet common in GCN classic, it makes sense to add in core schema itself.